lib/champaign_queue/clients/sqs.rb

```diff
@@ -0,0 +1,37 @@
+require 'aws-sdk'
+
+module ChampaignQueue
+  module Clients
+    class Sqs
+
+      class << self
+        def push(params)
+          new(params).push
+        end
+      end
+
+      def initialize(params)
+        @params = params
+      end
+
+      def push
+        return false unless queue_url
+
+        client.send_message({
+          queue_url:    queue_url,
+          message_body: @params.to_json
+        })
+      end
+
+      private
+
+      def client
+        @client ||= Aws::SQS::Client.new
+      end
+
+      def queue_url
+        ENV['SQS_QUEUE_URL']
```

Comment:
IMPORTANT CHANGE: By setting our sqs url to ENV, we save on one less http request per push, i.e. it's no longer necessary to call  `client.get_queue_url(opts).queue_url`.